client/public/locales/es/translation.json

```diff
@@ -54,9 +54,9 @@
   "Theorems": "Teoremas",
   "Not unlocked yet": "No desbloqueado aún",
   "Not available in this level": "No disponible en este nivel",
-  "A repository of learning games for the proof assistant <1>Lean</1> <i>(Lean 4)</i> and its mathematical library <5>mathlib</5>": "Un repositorio de juegos para aprender el asistente de demostración <1>Lean</1>, <i>(Lean 4)</i> y su biblioteca matemática <5>mathlib</5> ",
-  "No Games loaded. Use <1>http://localhost:3000/#/g/local/FOLDER</1> to open a game directly from a local folder.": "No se ha cargado ningún juego. Use <1>http://localhost:3000/#/g/local/FOLDER</1> para abrir un juego directamente desde una carpeta local",
-  "<0>If you are considering writing your own game, you should use the <1>GameSkeleton Github Repo</1> as a template and read <3>How to Create a Game</3>.</0><1>You can directly load your games into the server and play it using the correct URL. The <1>instructions above</1> also explain the details for how to load your game to the server. We'd like to encourage you to contact us if you have any questions.</1><p>Featured games on this page are added manually. Please get in contact and we'll happily add yours.</p>": "<0>Si está considerando escribir su propio juego, use el <1>GameSkeleton Github Repo</1> como plantilla y lea <3>How to Create a Game</3>.</0><1>Puede cargar directamente los juegos en el servidor y jugarlo usando la URL adecuada. Las <1>instrucciones anteriores</1> también explican los detalles sobre cómo cargar su juego en el servidor. Le animamos a ponerse en contacto con nosotros si tiene preguntas.</1><p>Los juegos incluidos en esta página son añadidos manualmente. Por favor, contactenos y añadiremos el suyo encantados.</p>",
+  "A repository of learning games for the proof assistant <1>Lean</1> <i>(Lean 4)</i> and its mathematical library <5>mathlib</5>": "Un repositorio de juegos educativos sobre el asistente de pruebas <1>Lean</1>, <i>(Lean 4)</i> y su librería matemática <5>mathlib</5> ",
```

Comment:
1) The original sentence was missing a preposition. I fixed that, as well as switching "games to learn..." to "educational games".
2) There isn't really a word for "proof assistant" in Spanish, but Wikipedia uses "asistente de pruebas" so I used that too.
3) In the programming sense, I've only ever heard the word "librería" for a library. Wikipedia disagrees with me here, so I'd like a second opinion.

Reply:
2. At least in Spain, I have always heard "asistente de demostración" (not "de pruebas").
3. Sometimes in Spain "biblioteca" is used in this context (some people have strong opinions about this, personally I don't).